README.md

```diff
@@ -2,142 +2,44 @@
 
 [![build status][build-badge]][build-href]
 
-Toolkit for component-oriented styling
+Styletron is an universal toolkit for component-oriented styling. It falls into the CSS in JS category. Styletron works great with React but can be used with other frameworks or plain JS as well.
 
-**Check our documentation at [styletron.org](https://www.styletron.org)**.
+**Go to our documentation at [styletron.org](https://www.styletron.org)**!
 
-[*Looking for v3.x docs?*](https://www.styletron.org/v3/) | [*v3.x to v4.x migration guide*](docs/v3-migration-guide.md)
-
-## Packages
-
-* [`styletron-engine-atomic`](packages/styletron-engine-atomic)
-* [`styletron-react`](packages/styletron-react)
-* [`styletron-standard`](packages/styletron-standard)
-
-## Design principles
```

Comment:
These aren't on the styletron.org site, are they?

Something like this I think is potentially valuable to keep in the README

Reply:
Yea, it's not exactly there or described differently. I put this section back.